notebooks/basics/multiple-systems.ipynb

```diff
@@ -804,59 +842,64 @@
    "id": "4f9e91e9",
    "metadata": {},
    "source": [
-    "#### Independent operations\n",
+    "#### Independent operations and the tensor product\n",
+    "\n",
+    "In the previous sections, we observed that a tensor product state $\\vert \\phi\\rangle \\otimes \\vert \\psi \\rangle$ on a joint system $(\\mathsf{X}, \\mathsf{Y})$ represents independence between the systems, roughly meaning that each system can be described in its own terms. We want to capture a similar notion of independence for operations $M$ and $N$ acting on systems $\\mathsf{X}$ and $\\mathsf{Y}$, respectively. More specifically, we'd like to capture the notion of \"$M$ acts on $\\mathsf{X}$ and $N$ acts on $\\mathsf{Y}$\" in terms of a single operation on the joint system $(\\mathsf{X}, \\mathsf{Y})$. It turns out that this notion can be expressed formally by the tensor product $M \\otimes N$ of the two operations. \n",
+    "\n",
+    "Suppose $(\\mathsf{X}, \\mathsf{Y})$ is in a tensor product state $\\vert \\phi\\rangle \\vert\\psi\\rangle$. If $M$ acts on $\\mathsf{X}$ and $N$ acts on $\\mathsf{Y}$, we should then expect the joint state to change as $M \\vert \\phi \\rangle N\\vert\\psi\\rangle$. Therefore, we define the tensor product operation $M \\otimes N$ to act on tensor product states in precisely this way.\n",
+    "\n",
+    "$$\n",
+    "  (M \\otimes N)\n",
+    "  \\vert \\phi \\rangle \\otimes \\vert \\psi \\rangle \n",
+    "  = M \\vert\\phi\\rangle \\otimes \n",
+    "  N \\vert\\psi\\rangle\n",
+    "$$\n",
     "\n",
-    "Now suppose that we have multiple systems and we *independently* perform *separate operations* on the systems.\n",
+    "This specification uniquely defines $M \\otimes N$ by linearity. Any state that is not a tensor product state can nevertheless be written as a linear combination of standard basis vectors, which are themselves tensor product states. The action of a linear operator on a basis uniquely defines that operator. \n",
```

Comment:
Let's not introduce the term "operator" at this point... "linear operator" -> either "linear mapping" or "matrix."